woo-razorpay.php

```diff
@@ -1846,7 +1851,7 @@ function razorpay_webhook_init()
 }
 
 define('RZP_PATH', plugin_dir_path( __FILE__ ));
-define('RZP_CHECKOUTJS_URL', 'https://checkout.razorpay.com/v1/checkout.js');
+define('RZP_CHECKOUTJS_URL', 'https://betacdn.np.razorpay.in/checkout/builds/branch-builds/1cc/kinsta-test-wc/v1/checkout.js');
```

Comment:
@poojarazor Please don't do this kind of mistake. Update it.

Reply:
got it